src/spikeinterface/exporters/to_phy.py

```diff
@@ -64,6 +65,8 @@ def export_to_phy(
         Dtype to save binary data
     verbose: bool
         If True, output is verbose
+    use_relative_path : bool, default: False
```

Comment:
Can you exapnd on this. When you say "saves the `dat_path` as a relative path, ... " 

I am kind of expecting that there is something after, relative to what?

Reply:
@h-mayorquin. Great point, that is a lazy docstring. I'll fix that shortly just got to lab so need to set up for the day.